### PR TITLE
TEST: Add unit tests for Models module and refactor all unit tests

### PR DESCRIPTION
**Pull request by caohhung**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+db_dir_path = os.path.dirname(os.path.realpath(__file__)) + '/.unittest'
+if not os.path.exists(db_dir_path):
+    os.mkdir(db_dir_path)
+db_file_path = db_dir_path + '/test_db.db'
+
+
+@pytest.fixture
+def app():
+    """Test fixture to set app config and remove previous test files."""
+    db.session.remove()
```
Comment (scattm) [inline tests/conftest.py]: I think it should be:
```
if not not db.session
    db.session.remove()
```

File: tests/test_models.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        assert not user_1.jobs
+
+        db.session.add(user_2)
+        db.session.commit()
+        assert user_2.get_id() == '2'
+        assert user_2.email == self.user_2['user_email']
+        assert user_2.password
+        assert user_2.is_active
+
+    @pytest.mark.skip(reason='Should improve exception')
```
Comment (scattm) [inline tests/test_models.py]: Missing docstring 😢

**Comment by scattm** [comment]
Great work! Thank Hung 😄

File: tests/conftest.py (hunk 1 above)
Comment (caohhung) [inline tests/conftest.py]: This part is beginning of test, so we don't care what kind of db.session is...Just say goodbye to it

**Pull request merged**